src/main/java/de/siegmar/logbackgelf/GelfLayout.java

```diff
@@ -215,7 +215,11 @@ private void addField(final Map<String, Object> dst, final String key, final Str
             addWarn("staticField key '" + key + "' is illegal. "
                 + "Keys must apply to regex ^[\\w.-]*$");
         } else {
-            dst.put(key, value);
+            if (value.startsWith("[")) {
+                dst.put(key, value.substring(1, value.trim().length() - 1).split(","));
+            } else {
+                dst.put(key, value);
+            }
         }
```

Comment:
Please also add tests for malformed values like `tags:[`, `tags:[abc`, `tags:[,`, `tags:[,]`, `tags:[]` that demonstrates expected behavior.